app.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const app = express();
-const {getTopics, getArticles, getArticleById, getArticleCommentById, postCommentByArticleId, patchArticleVote} = require('./controllers/con-topics')
+const {getTopics, getArticles, getArticleById, getArticleCommentById, postCommentByArticleId, patchArticleVote, getUsers} = require('./controllers/con-topics')
```

Comment:
As you've added more endpoints (dealing with articles and users) the name of this file `con-topics.js` isn't terrible accurate anymore, you could choose to split your controllers into separate files.